delivery_state/i18n/delivery_state.pot

```diff
@@ -101,8 +101,8 @@ msgid "In transit"
 msgstr ""
 
 #. module: delivery_state
-#: model:ir.model.fields.selection,name:delivery_state.selection__stock_picking__delivery_state__incidence
-msgid "Incidence"
+#: model:ir.model.fields.selection,name:delivery_state.selection__stock_picking__delivery_state__incident
+msgid "incident"
```

Comment:
Beware of manual changes:

```suggestion
msgid "Incident"
```

Reply:
Thanks. I tend to use the caps-wise replacement so either I didn't do it this time or it failed.